### DISCUSSION TITLE
AX Pro and use Diagnostic Variable

### DISCUSSION THREAD

**Discussion by PatriceHenriet** [petrleocompel/hikaxpro_hacs#103] [Q&A]
Hi

(beginning on home assistant)

For a device (Passive Infrared Detector or Smoke detector) appear diagnostic session Alarm (or battery, Armed, etc etc)., if alamr is on and Infrared Detector detect an human, the Alarm under Diagnotic switch to ON. Correct ?

And if yes, how to use this information under Automation?

Regards

**Comment by petrleocompel** [comment]
I do not think so. 
But what are you searching for is more like issue #2 
And we cannot currently detect and get real event data. 
Only solution is quick pull of data and I also do not think it is really a correct solution

**Reply by PatriceHenriet** [reply]
Ok i undestand, just want if smoke detector is on alarm, open the shutter, or if a outside infrared detector is on alarm, close the shutter.

The best way is use SIA but doesn't work between AX PRO and HA

Possible to reduce at 20s. ?

**Reply by petrleocompel** [reply]
@PatriceHenriet I know that some users reduced update intervals to 1s... I do not guarante functionality... But I guess it works...

**Comment by petrleocompel** [comment]
What do you mean by `SIA` ? @PatriceHenriet
